tests/test_mqtt_block.py

```diff
@@ -1,19 +1,75 @@
+from unittest.mock import patch, MagicMock
 from nio.block.terminals import DEFAULT_TERMINAL
 from nio.signal.base import Signal
 from nio.testing.block_test_case import NIOBlockTestCase
-from ..example_block import Example
+from ..mqtt_publish_block import MqttPublish
+from ..mqtt_subscribe_block import MqttSubscribe
 
+import paho.mqtt.client as mqtt
```

Comment:
Check out pep 8 here: https://www.python.org/dev/peps/pep-0008/#imports

```
Imports should be grouped in the following order:

standard library imports
related third party imports
local application/library specific imports
```
This tell me that `paho` should be after `unittest` but before the relative imports. I think right before the `nio` imports would be good. Could also put it after those.